src/components/protocol_handler/test/protocol_handler_tm_test.cc

```diff
@@ -385,6 +394,8 @@ class ProtocolHandlerImplTest : public ::testing::Test {
 
   testing::NiceMock<MockProtocolHandlerSettings> protocol_handler_settings_mock;
   std::shared_ptr<ProtocolHandlerImpl> protocol_handler_impl;
+  std::shared_ptr<testing::NiceMock<MockServiceStatusUpdateHandlerListener> >
```

Comment:
@ValeriiMalkov that name looks long. maybe we can use some typedef here?

Reply:
@mked-luxoft Fixed in commit 46d5aa36222d24983af116636edf39b0dee4e979